plugin/core/views.py

```diff
@@ -976,44 +975,36 @@ def _format_diagnostic_related_info(
     location = info["location"]
     return '<a href="{}">{}</a>: {}'.format(
         location_to_href(config, location),
-        location_to_human_readable(config, base_dir, location),
-        info["message"]
+        text2html(location_to_human_readable(config, base_dir, location)),
```

Comment:
is there a need to call text2html for this place, the location_to_human_readable will return a path?